Vindicta.Altis/Templates/Loadouts/CUP_RUS_CIVILIAN/CUP_PLAYER_1.sqf

```diff
@@ -0,0 +1,77 @@
+removeAllWeapons this;
+removeAllItems this;
+removeAllAssignedItems this;
+removeUniform this;
+removeVest this;
+removeBackpack this;
+removeHeadgear this;
+removeGoggles this;
+
+[this, selectRandom gVanillaFaces, "ace_novoice"] call BIS_fnc_setIdentity;
+
+// ==== Headgear ====
+if (random 10 < 4) then {
+	private _headgear = [
+		"CUP_H_FR_BandanaGreen",
+		"CUP_H_FR_BandanaWdl",
+		"CUP_H_PMC_Beanie_Black",
+		"CUP_H_SLA_BeanieGreen",
+		"CUP_H_PMC_Beanie_Khaki",
+		"CUP_H_C_Beanie_02",
+		"CUP_H_C_Beanie_04"
+	];
+
+	this addHeadgear selectRandom _headgear;
+};
+
+//	==== Uniform ====
+private _uniforms = [
+	"CUP_U_C_Worker_01",
+	"CUP_U_C_Woodlander_01",
+	"CUP_U_C_Villager_01",
+	"CUP_U_C_Functionary_jacket_01",
+	"CUP_U_C_Suit_01",
+	"CUP_U_C_Rocker_01",
+	"CUP_U_C_Priest_01",
+	"CUP_U_C_Citizen_01",
+	"CUP_U_C_Mechanic_01",
+	"CUP_U_C_racketeer_01",
+	"CUP_U_C_Profiteer_01",
+	"CUP_U_O_CHDKZ_Lopotev"
+];
+
+this forceAddUniform selectRandom _uniforms;
+this addItemToUniform "ACE_Flashlight_Maglite_ML300L";
+
+// ==== Backpack ====
```

Comment:
Player doesn't get a backpack at spawn

Reply:
We might change it, but until we do all templates should follow the same spawn kit.